superbuild/CMakeLists.txt

```diff
@@ -37,6 +37,13 @@ set(CCACHE_DIR "default" CACHE STRING "ccache cache directory")
 set(PREBUILT_STAGE2 "NO" CACHE STRING "Provide pre-built stage2 archive to skip pre ispc-stage2 jobs")
 set(PREBUILT_STAGE2_PATH "NO" CACHE STRING "Provide pre-built stage2 toolchain path")
 set(STAGE2_TOOLCHAIN_INSTALL_PREFIX "NO" CACHE STRING "Path to install stage2 Clang/LLVM to")
+set(JOBS "" CACHE STRING "Set the number of jobs for internal build commands (ninja)")
+set(CMAKE_BUILD_TYPE "Release" CACHE STRING "Set the build type for builds (overrides per-component build type switches)")
```

Comment:
Have you tested it? As far as I understand how CMake works, redefining `CMAKE_*` variables work only when done before the `project(...)` directive, otherwise CMake silently discards this redeclaration. And it makes sense, as at the point of `project(...)` directive CMake needs to know about key parameters for the project, so hence changing them in the middle of the project is not supported.

That's another reason why redefining `CMAKE_*` variable or duplicating its functionality is typically a bad idea and may strike back in unexpected way.

Reply:
That's ok - we can delete the setter line. The logic controlling the type for sub-components is the main use for this change. I wasn't aware about that for Windows. If we want to prevent the user from mixing types, we should remove the sub-component variables, but we should still pass the build type along. Currently, it's hard-coded to Release.